zowe/zowe-cli
# Migrate pr1840

**What It Does**


This is the first PR of many more to come in before the Zowe V3 GA date.
The intention is to migrate the changes done in the following PR:
- #1840

**How to Test**


- Verify that the pipeline can go for longer than 75 minutes
- Verify that the Sonar reporter is successfully reporting based on test results

In other words, let's give the pipeline some time to complete. 😋 

**Review Checklist**
I certify that I have:

**Additional Comments**


I wanted to have this as an example of a clean migration from `master` to `next`.
The will be PRs where conflicts between the two branches will arise, so be cautious during any and all migration PRs 😋
